aws/resource_aws_guardduty_member_test.go

```diff
@@ -121,7 +147,7 @@ func testAccCheckAwsGuardDutyMemberExists(name string) resource.TestCheckFunc {
 	}
 }
 
-func testAccGuardDutyMemberConfig_basic(accountID, email string) string {
+func testAccGuardDutyMemberConfig_basic1(accountID, email string) string {
```

Comment:
We should keep the old name here and rename the new config `_invite` or something.